src/Categories/Bicategory/LocalCoequalizers.agda

```diff
@@ -0,0 +1,46 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Bicategory
+
+module Categories.Bicategory.LocalCoequalizers {o ℓ e t} (𝒞 : Bicategory o ℓ e t)  where
+
+open import Categories.Diagram.Coequalizer
+open import Level
+open import Categories.Functor.Properties
+import Categories.Bicategory.Extras as Bicat
+open Bicat 𝒞
+open import Categories.Functor
+
+postCompFunc : {A B C : Obj} → B ⇒₁ C → Functor (hom A B) (hom A C)
```

Comment:
If this is "just" a synonym for `_⊚-`, why give it a new, much longer name? I don't quite see the reason for that.